cmd/csi-sanity/sanity_test.go

```diff
@@ -45,7 +45,8 @@ func TestSanity(t *testing.T) {
 		return
 	}
 	if len(endpoint) == 0 {
-		t.Fatalf("--%s.endpoint must be provided with an CSI endpoint", prefix)
+		t.Fatalf("--%sendpoint must be provided with an CSI endpoint", prefix)
```

Comment:
It needs to be `%s.endpoint` because all help has a period. For example there is `ginkgo.XXX` and `test.XXX`